### PR TITLE
py-pytorch-lightning: add +extra variant

### PR DESCRIPTION
**Pull request by adamjstewart**
Successfully builds on macOS 10.15.7 with Python 3.10.8 and Apple Clang 12.0.0.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-onnx-runtime/package.py [hunk 1]
```diff
@@ -14,13 +14,16 @@ class PyOnnxRuntime(CMakePackage, PythonExtension):
     stays up to date with the ONNX standard with complete
     implementation of all ONNX operators, and supports all
     ONNX releases (1.2+) with both future and backwards
-    compatibility."""
+    compatibility.
+
+    Deprecated in favor of py-onnxruntime.
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-onnx-runtime/package.py:19]: Both the [GitHub](https://github.com/microsoft/onnxruntime) and [PyPI](https://pypi.org/project/onnxruntime/) are named onnxruntime, not onnx-runtime.

**Comment by adamjstewart** [comment]
@tldahlgren I don't think it makes sense to assign me to my own PR 😆

**Comment by tldahlgren** [comment]
> @tldahlgren I don't think it makes sense to assign me to my own PR 😆

🤦‍♀️

**Review by tldahlgren — APPROVED** [review]
Confirmed new/modified sha256 and tags.

**Pull request merged**